packages/node-core/src/utils/string.ts

```diff
@@ -0,0 +1,13 @@
+// Copyright 2020-2024 SubQuery Pte Ltd authors & contributors
+// SPDX-License-Identifier: GPL-3.0
+
+export function handledStringify(obj: any): string {
+  try {
+    return JSON.stringify(obj);
+  } catch (error) {
+    if (error instanceof Error) {
+      return error.stack || error.message;
+    }
+    return 'Unknown error when Stringify';
```

Comment:
Can we give a hint as to where this could have gone wrong? It will make things harder to debug otherwise